docs/en_US/Tutorial/Nnictl.md

```diff
@@ -531,6 +528,62 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl experiment import [experiment_id] -f experiment_data.json
     ```
 
+* __nnictl experiment save__
+  * Description
+
+    Save nni experiment metadata and code data.
+
+  * Usage
+
+    ```bash
+    nnictl experiment save [OPTIONS]
+    ```
+
+  * Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------ |------|
+  |id|  True| |The id of the experiment you want to save|
+  |--path, -p|  False| |the folder path to store nni experiment data, default current working directory|
+  |--saveCodeDir, -s| | |save codeDir data of the experiment, default False|
+
+  * Examples
+
+  > save an expeirment
+
+  ```bash
+  nnictl experiment save [experiment_id] --saveCodeDir
+  ```
+
+* __nnictl experiment load__
+  * Description
+
+    Load a nni experiment.
+
+  * Usage
+
+    ```bash
+    nnictl experiment load [OPTIONS]
+    ```
+
+  * Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------ |------|
+  |--path, -p|  True| |the full file path of nni package|
+  |--codeDir, -c| True| |the path of codeDir|
```

Comment:
could add more description for `codeDir`. for example, "the path that you want to put the code of the saved experiment into". BTW, what if the saved experiment does not include code?

Reply:
updated, if the experiment package does not code files, NNI will do nothing, just use new codeDir to replace old one.